clocksilencer.ino

```diff
@@ -39,6 +41,14 @@ void loop() {
   mySec = now.second();
   myMin = now.minute();
   myHour = now.hour();
+Serial.print("It is ");
```

Comment:
This is to print the time. We have something similar after `Serial.println("RTC updated");`.

Maybe it's good moment to create a function of:

```c++
void printTime(DateTime now) {
    // ...
```

what do you think?

Reply:
I had similar idea, this may be worth doing.